constants/app.js

```diff
@@ -11,13 +11,11 @@ export const TRANSIFEX_BLACKLIST = [
 ];
 
 export const USERREPORT_BLACKLIST = [
```

Comment:
Let's remove this constant if we are not using it anymore.

Reply:
@pablopareja I did it myself (https://github.com/resource-watch/resource-watch/pull/1032/commits/f68a8fc12b7b83ead081513a26647e9c98bd8ab1), please check and merge if everything is fine 👍